.github/workflows/bootzooka-deploy.yml

```diff
@@ -15,37 +14,8 @@ on:
       - "helm/**"
 
 jobs:
-  test:
-    name: Test
-    runs-on: ubuntu-18.04
-
-    steps:
-    - name: Check-out repository
-      id: repo-checkout
-      uses: actions/checkout@v2
-
-    - name: Set up JDK 11
-      id: jdk-setup
-      uses: actions/setup-java@v1
-      with:
-        java-version: 11
-
-    - name: Cache SBT
-      id: cache-sbt
-      uses: actions/cache@v2
-      with:
-        path: |
-          ~/.sbt
-          ~/.ivy2/cache
-          ~/.coursier
-        key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt') }}
-
-    - name: Run tests
-      id: run-tests
-      run: sbt test
-
   deploy:
-    if: github.event_name != 'pull_request' && startsWith(github.ref, 'refs/tags/v')
```

Comment:
I guess we don't want to run deploy on PRs?

Reply:
I removed PR trigger above in the file 
![image](https://user-images.githubusercontent.com/59844835/144679868-ab3b69b9-ff7d-40f9-af95-eeeddebbe3f3.png)

Reply:
startsWith condition should be ok as for now

Reply:
ah ok makes sense :)